bee-hive/bee_hive/__init__.py

```diff
@@ -1,6 +1,6 @@
-
 __all__ = [
-    "Agent",
-    "Workflow",
-    "Interface",
+    "agent",
```

Comment:
current code is:
```
 from .workflow import Workflow
 from .bee_agent import BeeAgent
 from .agent import save_agent, restore_agent, remove_agent
```

_all__ can limit what is exported when doing an import *

May be multiple ways for this to work, running demos will probably fail if this change isn't sufficient (needs testing)